src/sass/_forms.scss

```diff
@@ -25,13 +25,13 @@
     display: flex;
 
     .form-navigation {
-      min-width: 13rem;
+      min-width: 14rem;
       overflow: auto;
 
       .p-side-navigation__link {
         justify-content: flex-start;
         margin-bottom: 0;
-        min-width: 13rem;
+        min-width: 14rem;
```

Comment:
The link width shouldn't grow or we get again the vertical scroller:
![image](https://github.com/canonical/lxd-ui/assets/1155472/28ee4f8d-35c3-4693-be7b-866572ab8ce2)

Reply:
Yeah I see, do you mean the horizontal scroller shouldn't show right? Adjusting the link min-width affects the horizontal spacing?

Reply:
We increased `form-navigation` to avoid the vertical scroller in the menu.

Here we increase its child `p-side-navigation__link`. This recreates the issue. Hence, we shouldn't change the `p-side-navigation__link` and everything should be fine.

Reply:
cool made the change